packages/zapp/console/src/components/flytegraph/ReactFlow/ReactFlowGraphComponent.tsx

```diff
@@ -103,6 +125,13 @@ const ReactFlowGraphComponent = (props) => {
     }));
   }, [onNodeSelectionChanged]);
 
+  useEffect(() => {
+    setState((state) => ({
+      ...state,
+      onMapTaskSelectionChanged: onMapTaskSelectionChanged,
+    }));
+  }, [onMapTaskSelectionChanged]);
```

Comment:
nit: do we need the two useEffects that take onMapTaskSelectionChanged and onNodeSelectionChanged as dependencies (L:126 & L:133)?

Reply:
good point! merging them